src/components/ClusterOverview/Capacity/Capacity.js

```diff
@@ -15,70 +17,103 @@ const pfGetUtilizationDonutTooltipContents = d =>
     d[0].name
   }</span>`;
 
-class CapacityBody extends React.PureComponent {
-  render() {
-    const { stats } = this.props;
-    return (
-      <div className="kubevirt-capacity__items">
-        {Object.keys(stats).map(key => {
-          const available = stats[key].data.total - stats[key].data.used;
-          const columns = [['Used', stats[key].data.used], ['Available', available]];
-          const sum = columns.reduce((acc, x) => acc + x[1], 0);
-          return (
-            <div key={key} className="kubevirt-capacity__item">
-              <h2 className="kubevirt-capacity__item-title">{stats[key].title}</h2>
-              <h6>
-                {`${Number.isInteger(available) ? available : available.toFixed(1)}${
-                  stats[key].unit
-                } available out of ${stats[key].data.total}${stats[key].unit}`}
-              </h6>
-              <div>
-                <DonutChart
-                  id={`donut-chart-${key}`}
-                  size={{ width: 150, height: 150 }}
-                  data={{
-                    unload: true,
-                    columns,
-                    order: null,
-                  }}
-                  unloadBeforeLoad
-                  tooltip={{ contents: pfGetUtilizationDonutTooltipContents }}
-                  title={{
-                    primary: `${Math.round((100 * columns[0][1]) / sum).toString()}%`,
-                    secondary: columns[0][0],
-                  }}
-                />
-              </div>
-            </div>
-          );
-        })}
+const CapacityItem = ({ idSuffix, title, used, total, unit }) => {
+  const available = total - used;
+  const columns = [['Used', used], ['Available', available]];
+
+  const sum = columns.reduce((acc, x) => acc + x[1], 0);
```

Comment:
isn't the sum equal to total?

Reply:
good point. Moving code here and there ...